configs/config-test.json

```diff
@@ -43,7 +43,8 @@
 			"keystorePassword": "",
 			"catServerHost": "",
 			"catServerPort": 443,
-			"verticleInstances": 2
+			"verticleInstances": 2,
+			"basePath": "/ngsi-ld/v1"
```

Comment:
base path is removed from config-test in the latest commit [here](https://github.com/datakaveri/iudx-data-ingestion-server/pull/63/commits/83f4efb5e47378d1540a8d578bc7d09517382e0d). It could be configured from config-dev